### DISCUSSION TITLE
Is it possible to use both, quasar and primevue?

### DISCUSSION THREAD

**Discussion by froschi123** [quasarframework/quasar#14467] [General - Components / Directives / etc]
HI, i am using Quasar and build a framework for it in our company. But there are some components in primevue, that i would like to use as well. e.g. the DataTable component of primevue.

Is this possible, and if so, how would i achieve the combination?
Thanks, Bernd

**Comment by metalsadman** [comment]
try it, depends on the css namespaces, there might be some clashes.

**Comment by WazeJinx** [comment]
@froschi123 Did you find a way to import PrimeVue components styling into quasar without CSS problems?

**Comment by sjoshid** [comment]
The absence of robust filtering options in QTable is genuinely disappointing. While it's possible to implement custom filtering from the ground up, as I did, the header inputs tend to appear bulky and unrefined—even when using the dense layout. Supporting filtering for various data types, such as dates or unique values, proves especially challenging. Given that table filtering is a common expectation from a modern UI library, stronger built-in support seems reasonable.

To be clear, I appreciate many aspects of Quasar and have worked extensively with it. My frustration only emerged after confronting the table filtering limitations.
